plans/data-transfer-seeds-leeches/README.md

```diff
@@ -0,0 +1,53 @@
+# `Plan:` Data Transfer of combinations of Seeds and Leeches (Bitswap/GraphSync)
+
+Create an environment in which combinations of seeds and leeches are varied. This test is not about content discovery or connectivity, it is assumed that all nodes are connected and that these are executed in an homogeneous network (same CPU, Memory, Bandwidth).
```

Comment:
Do we plan on running fully fledged IPFS nodes, or do we want to isolate Bitswap by instantiating programmatically?

Reply:
I think it would be best to run IPFS nodes, to more closely simulate a real world scenario

Reply:
There is:

```
» ipfs stats --help
USAGE
  ipfs stats - Query IPFS statistics.

SYNOPSIS
  ipfs stats

DESCRIPTION

  'ipfs stats' is a set of commands to help look at statistics
  for your IPFS node.

SUBCOMMANDS
  ipfs stats bitswap - Show some diagnostic information on the bitswap agent.
  ipfs stats bw      - Print ipfs bandwidth information.
  ipfs stats repo    - Get stats for the currently used repo.

  For more information about each command, use:
  'ipfs stats <subcmd> --help'
```

and 

```
» ipfs bitswap stat --help
USAGE
  ipfs bitswap stat - Show some diagnostic information on the bitswap agent.

SYNOPSIS
  ipfs bitswap stat [--verbose | -v] [--human]

OPTIONS

  -v, --verbose  bool - Print extra information.
  --human        bool - Print sizes in human readable format (e.g., 1K 234M 2G).
```